ui/src/lib/resultsList/ResultsList.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import SvelteTable from 'svelte-table/src/index';
+	import { resultsData } from './data';
+
+	const rows = resultsData;
+
+	const columns = [];
+	if (resultsData.length > 0) {
+		const prepareColumns = {};
+		const object = resultsData[0];
+		for (const key in object) {
+			if (Object.prototype.hasOwnProperty.call(object, key)) {
+				prepareColumns[key] = {
+					key: key,
+					title: key.toLocaleUpperCase(),
+					value: (v) => v[key],
+					sortable: true
+				};
+			}
+		}
+
+		prepareColumns['url'].renderValue = (v) =>
+			`<img src="https://picsum.photos/seed/${v.name}/20/20" alt="icon" width=20 height=20 >`;
+		prepareColumns['url'].title = 'IMAGE';
+
+		for (const key in prepareColumns) {
+			columns.push(prepareColumns[key]);
+		}
+	}
+
+	const exludeColumns = ['url'];
+	let search: string = '';
+	$: visibleRows = search
+		? rows.filter((el) => {
+				for (const key in el) {
+					if (exludeColumns.includes(key)) continue;
+
+					if (Object.prototype.hasOwnProperty.call(el, key)) {
+						const element: string | number = el[key];
+						const include = element.toString().toLowerCase().includes(search.toLowerCase());
+						if (include) return true;
+					}
+				}
+				return false;
+		  })
+		: rows;
+
+	let hiddenColumns = [];
+	$: visibleColumns = columns.filter((el) => !hiddenColumns.includes(el.key));
+</script>
+
+<label>Search in resluts: <input bind:value={search} /></label>
+<div>
+	<p>Hide columns</p>
```

Comment:
I think if we limit the number of colums to minimum then hiding columns will not be needed.
Also selecting checkbox to check which columns to hide isn't UX friendly.

Reply:
![obraz](https://user-images.githubusercontent.com/26521377/160283690-7fb36664-e286-414a-a6de-b8b5eff9c593.png)
This is in task